libpyvinyl/DiffractionData/api_spec_v2.py

```diff
@@ -0,0 +1,74 @@
+"""
+The following documents how the various classes in libpyvinyl and a IO class to be implemented
+are supposed to be used.
+"""
+
+import InteractionCalculator, DiffractionCalculator, DataCollection
+import WavefrontData, SampleData
+
+
+
+base_path = 'my_base'
+interaction = InteractionCalculator()
+interaction.base_path = base_path
```

Comment:
is base path the common root of all simulation output or just for the interaction output? Is it a directory or a file? All this needs to be made clear and tested against.

Reply:
It's the common root directory of all simulation output. It's proposed to be needed for defining an `instrument`. Would `base_dir` be a better name?
@shervin86 may comment on this?